<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preparation</class>
 <widget class="QWidget" name="preparation">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,1">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Load Project</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton">
             <property name="text">
              <string>Load Project</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Project Dir</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="project_line"/>
             </item>
             <item>
              <widget class="QPushButton" name="peoject_button">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>Create a New Project</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,1">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Job Dir</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="job_line"/>
             </item>
             <item>
              <widget class="QPushButton" name="job_button">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Load Protein</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Portein File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="protein_file"/>
             </item>
             <item>
              <widget class="QPushButton" name="protein_button">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>If the protein is upprepared, you can do:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="unprepare_button">
               <property name="text">
                <string>Protein Preparation</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Load Ligands</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Lignad Files</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ligand_button">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2,1">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Affinity Data</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="affinity_line"/>
             </item>
             <item>
              <widget class="QPushButton" name="button_dGImport">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_5">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Alignment</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,0">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Identify Crystal Structures</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Please identify on the right</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Ligand Alignment</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="align_box">
               <item>
                <property name="text">
                 <string>Constrained Docking</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>None</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="hermite_button">
             <property name="text">
              <string>Align and Show on Hermite</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>If you are not satisfied with the result, 
you can do overlay one more time.</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="right_panel" native="true">
       <layout class="QVBoxLayout" name="table_layout"/>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_next_prep">
     <property name="text">
      <string>Next-&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
